seerpy/seerpy.py

```diff
@@ -1173,14 +1176,22 @@ def get_diary_labels(self, patient_id, label_type='all', offset=0, limit=100, fr
                 labels = group['labels']
 
                 if not labels:
+                    label_results = response
```

Comment:
Wouldn't this have undesirable results if for instance the first labelGroup has labels, but the second labelGroup doesn't? i.e. wouldn't it "reset" the `label_results` variable?

```suggestion
                    continue
```